src/view-slot.js

```diff
@@ -209,7 +209,7 @@ export class ViewSlot {
   * @param skipAnimation Should the removal animation be skipped?
   * @return May return a promise if the view removal triggered an animation.
   */
-  remove(view: View, returnToCache?: boolean, skipAnimation?: boolean): void | Promise<View> {
+  remove(view: View, returnToCache?: boolean, skipAnimation?: boolean): void | Promise<View> | void{
```

Comment:
If this PR is getting merged, lets fix the return type of `.remove`, it should be the same as of `.removeAt`.

Reply:
Done.